test/functional/test_site_algolia_plugin/index.md

```diff
@@ -13,7 +13,7 @@
   <li><a class="dropdown-item" href="/">Two</a></li>
 </dropdown>
 
-## Modals should have algolia-no-index class
+## Modals should have algolia-no-index class but triggers should not
 
 <modal title="Modal" id="modal:trigger_id">
```

Comment:
We could specify in the modals/triggers and popovers/questions themselves that they should or should not have `agolia-no-index`? For example:

```html
<popover effect="fade" title="Title" placement="top">
  <div slot="content">Content `<div>` **should** have `agolia-no-index`</div>
  <button class="btn btn-secondary">Popover trigger **should not** have `agolia-no-index`</button>
</popover>
```

Reply:
That's a good point! I've updated the contents to reflect this.